marimbabot_audio/scripts/onset_detection.py

```diff
@@ -520,6 +526,6 @@ def publish_cqt(self, cqt):
         self.pub_cqt.publish(msg)
 
 if __name__ == '__main__':
-    rospy.init_node("detect_onset")
+    rospy.init_node("detect_onset",log_level=rospy.DEBUG)
```

Comment:
Why is the loglevel hardcoded to debug?

Reply:
If you want to turn on the dedug info, you can choose not to output to screen in launch file, right? or what's your opionin?

Reply:
Exactly. ROS has multiple ways of setting the log level, but I would not hardcode it.